lib/webaction.js

```diff
@@ -90,7 +90,8 @@ function getParams(params) {
         newRelicEventsURL: params.newRelicEventsURL,
         newRelicApiKey: params.newRelicApiKey,
         times: params.times,
-        customWorker: true
+        customWorker: true,
+        predictedRunDuration: params.predictedRunDuration
```

Comment:
do we want/need predicted run duration in custom workers? IMO we don't need it because we will never be able to predict or control a custom worker time length/cost as it can change without us knowing.

also, just realized, we don't have a cost value/duration for custom workers in core cost values. I think we should have static values for these for all custom workers. this will be inaccurate of course since custom workers can choose their own memory/timeout values but we need to have something thats onthe higher end. wdyt